control/pure_pursuit/CMakeLists.txt

```diff
@@ -3,6 +3,8 @@ project(pure_pursuit)
 
 if(NOT CMAKE_CXX_STANDARD)
 	set(CMAKE_CXX_STANDARD 14)
+        set(CMAKE_CXX_STANDARD_REQUIRED ON)
+        set(CMAKE_CXX_EXTENSIONS OFF)
```

Comment:
Could you please fix the indenting here?

Reply:
fixed!